function/expression.go

```diff
@@ -19,24 +22,26 @@ import (
 // * Contains current timerange being queried for - this can be
 // changed by say, application of time shift function.
 type EvaluationContext struct {
-	TimeseriesStorageAPI  api.TimeseriesStorageAPI // Backend to fetch data from
-	MetricMetadataAPI     api.MetricMetadataAPI    // Api to obtain metadata from
-	Timerange             api.Timerange            // Timerange to fetch data from
-	SampleMethod          api.SampleMethod         // SampleMethod to use when up/downsampling to match the requested resolution
-	Predicate             predicate.Predicate      // Predicate to apply to TagSets prior to fetching
-	FetchLimit            FetchCounter             // A limit on the number of fetches which may be performed
-	Cancellable           api.Cancellable
+	TimeseriesStorageAPI  timeseries.StorageAPI   // Backend to fetch data from
+	MetricMetadataAPI     metadata.MetricAPI      // Api to obtain metadata from
+	Timerange             api.Timerange           // Timerange to fetch data from
+	SampleMethod          timeseries.SampleMethod // SampleMethod to use when up/downsampling to match the requested resolution
+	Predicate             predicate.Predicate     // Predicate to apply to TagSets prior to fetching
+	FetchLimit            FetchCounter            // A limit on the number of fetches which may be performed
+	Timeout               tasks.Timeout
 	Registry              Registry
 	Profiler              *inspect.Profiler // A profiler pointer
-	EvaluationNotes       *EvaluationNotes  //Debug + numerical notes that can be added during evaluation
-	UserSpecifiableConfig api.UserSpecifiableConfig
+	EvaluationNotes       *EvaluationNotes  // Debug + numerical notes that can be added during evaluation
+	UserSpecifiableConfig timeseries.UserSpecifiableConfig
 }
 
+// EvaluationNotes holds notes that occurred during evaluation.
```

Comment:
Maybe `notes that occurred during evaluation.` -> `notes that were recorded during evaluation.`?